Cargo.toml

```diff
@@ -90,9 +90,11 @@ del-geo = "=0.1.29"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
 # see https://github.com/emilk/egui/issues/4270
-egui-winit = "0.29"
+egui-winit = { version="0.29", default-features=false, features=["links"] }
 
 [target.'cfg(target_arch = "wasm32")'.dev-dependencies]
+# see https://github.com/emilk/egui/issues/4270
+egui-winit = { version="0.29", default-features=false, features=["links"] }
```

Comment:
it's now the same dependency between wasm and normal, can we move it into a common block of dev dependencies?